GroupIS1024/Pages/Index.cshtml

```diff
@@ -61,19 +61,19 @@
 
     <!-- Features -->
     <div class="features">
-        <h1>One place to tracl your team and more</h1><br />
+        <h1>One place to tracK your team and more</h1><br />
 
         <div class="row">
 
             <div class="feature-item col-lg-4 col-md-4 icon">
-                <a href="#payroll">
+                <a href="teams">
```

Comment:
The # indicates an internal page link.  Do these still work with that removed?  It appears that this is changing functionality.